### PR TITLE
Add some new useful ui classes

### PR DESCRIPTION
**Pull request by brettwold**
Moved from FPS config provider
Update libs and gradle builde

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,18 +1,29 @@
...
+      at: ~/
+
 reusable-steps:
   - &restore_cache
     restore_cache:
       keys:
-      - v1-dependencies-{{ checksum "build.gradle" }}
-      # fallback to using the latest cache if no exact match is found
-      - v1-dependencies-
+      - v1-dependencies-{{ checksum "dependencies.gradle" }}
```
Comment (louis-barman) [inline .circleci/config.yml]: Should be `v2-dependencies` according to Brett!

Comment (brettwold) [reply .circleci/config.yml]: Only if I want to clear the cache. Which I don't right now for this repo

File: ui-library/src/main/java/com/aevi/ui/library/views/SettingSwitch.java [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    }
+
+    private void init(Context context, AttributeSet attrs) {
+        changedSubject = PublishSubject.create();
+        View v = View.inflate(context, R.layout.setting_switch, this);
+        ButterKnife.bind(this, v);
+
+        if (attrs != null) {
+            TypedArray a = context.obtainStyledAttributes(attrs, R.styleable.SettingSwitch, 0, 0);
+            title.setText(a.getString(R.styleable.SettingSwitch_switchTitle));
```
Comment (GregHib) [inline ui-library/src/main/java/com/aevi/ui/library/views/SettingSwitch.java]: PascalCase, camelCase and underscore_case all in one name?

Comment (brettwold) [reply ui-library/src/main/java/com/aevi/ui/library/views/SettingSwitch.java]: Yeap you can blame Android for that... these are generated names

**Pull request merged**